src/frontends/pytorch/src/op/fft.cpp

```diff
@@ -28,18 +30,55 @@ namespace op {
 
 using namespace ov::op;
 
-OutputVector translate_fft_rfftn(const NodeContext& context) {
-    // aten::fft_rfftn(Tensor self, int[1]? s=None, int[1]? dim=None, str? norm=None) -> Tensor
-    num_inputs_check(context, 1, 4);
-    auto input = context.get_input(0);
-
-    auto const_neg_1 = context.mark_node(v0::Constant::create(element::i32, Shape{}, {-1}));
+namespace {
+Output<Node> normalize(const NodeContext& context,
+                       const Output<Node>& node,
+                       const Output<Node>& s,
+                       const std::string& norm,
+                       bool inverse) {
+    if (norm == "backward") {
+        // No normalization
+        return node;
+    }
+    // Apply normalizations
     auto const_0 = context.mark_node(v0::Constant::create(element::i32, Shape{}, {0}));
-    auto const_1 = context.mark_node(v0::Constant::create(element::i32, Shape{}, {1}));
+    auto n_int = context.mark_node(std::make_shared<v1::ReduceProd>(s, const_0));
+    auto n = context.mark_node(std::make_shared<v1::ConvertLike>(n_int, node));
+    Output<Node> normalized;
+    if (norm == "forward") {
+        // Normalize by 1/n
+        if (inverse) {
+            normalized = context.mark_node(std::make_shared<v1::Multiply>(node, n));
+        } else {
+            normalized = context.mark_node(std::make_shared<v1::Divide>(node, n));
+        }
+    } else if (norm == "ortho") {
+        // Normalize by 1/sqrt(n)
+        auto sqrt_n = context.mark_node(std::make_shared<v0::Sqrt>(n));
+        if (inverse) {
+            normalized = context.mark_node(std::make_shared<v1::Multiply>(node, sqrt_n));
+        } else {
+            normalized = context.mark_node(std::make_shared<v1::Divide>(node, sqrt_n));
+        }
+    } else {
+        FRONT_END_THROW("Unrecognized normalization mode. Only forward, backward and ortho are supported.");
```

Comment:
could you please include info about provided norm argument into error message?
it helps in debugging in future.

also error message includes backward as supported, but I do not see any checking for this mode here. Does it made intentionally?